conf/defaults.ini

```diff
@@ -773,3 +773,17 @@ license_path =
 [feature_toggles]
 # enable features, separated by spaces
 enable =
+
+[preferences]
```

Comment:
Agree this makes most sense, but if we're going this way we should add `default_theme` here as well with a fallback to legacy setting to make it consistent.

Reply:
yes, agree.

Reply:
changed this section to be only about date formating